### PR TITLE
Capstone Project for The Portal Module(ONE)

### PR DESCRIPTION
**Pull request by carlylechia**
This project was built to attest to the completion of the HTML and CSS module. Following the project requirements, I completed the following tasks and milestones:
 - Recorded a [loom video presentation of my project](https://www.loom.com/share/8ac3176a9f0a4cf0b0cadfaacb2ca02b)
 - Wrote a descriptive readme which contains almost all the basic information about the project
 - Setup linters for my project
 - Built and deployed the Home and about pages of the mobile version, while making sure there are no linters errors
 - Made the two pages responsive to larger screens
 - Used javascript and DOM manipulation to load the judges dynamically
 - Implemented the mobile menu and the more buttons with javascript
 - Also used transitions and animations to make the pages more user-friendly.
 
You can watch the loom video above to have a basic summary of the project.

### CODE REVIEW

File: stylesheets/about.css [hunk 1]
```diff
@@ -1 +1,163 @@
-/* This is an empty file */
+/* This file contains all the styling for the landing page for the about page of The VSD project */
+
+/* The mobile-first approach was used in styling. */
```
Comment (Whoistolu) [inline stylesheets/about.css]: - [ ] [OPTIONAL] Kindly remove all unused code so we can have a clean code-base. It is a good practice to have a clean code-base.

File: README.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ This project was deployed on github pages. [This is the link](https://carlylechi
 
 ### Documentation
 
-I have recorded [a loom video](https://www.loom.com/share/8ac3176a9f0a4cf0b0cadfaacb2ca02b) briefly throwing some light on the project.
+I have recorded [a loom video](https://www.loom.com/share/8ac3176a9f0a4cf0b0cadfaacb2ca02b) briefly throwing some light on the project. Feel free to check it out.
```
Comment (Whoistolu) [inline README.md:48]: - [x] [REQUIRED] Kindly fix the live version as the link seems broken. [Live Version](https://carlylechia.github.io/Vision-Straight/index.html)
It is important to deploy tour projects in order for other users to have access to it online.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+      </div>
+      <div class="landing-body">
+        <p>The Vision Straight Discovery (VSD) edition #1 2022 is a first of many shows to come, which aims at providing talented individuals a platform to showcase their musical talents, for a chance to become part of the next generation of superstars, and to be signed into the Vision Straight Records Brand, with loads of added advantages.</p>
+      </div>
+      <div class="landing-description">
+        <h2>2022.05.15(SUN) ~ 22(SUN)</h2>
+        <p>@ VS Concert Hall, Bamenda, Cameroon.</p>
+      </div>
+    </section>
+  </header>
```
Comment (Whoistolu) [inline index.html:77]: - [ ] [REQUIRED] In order to have a better user experience, kindly adjust the circular background image as it is currently obstructing the text.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
 |

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @carlylechia,

You have done a great job so far 👍🏽 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

To Highlight clap
✔️ Descriptive PR
✔️ Good README
✔️ Meaningful commit messages

_Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: script.js [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+  if (mobileNav.style.display === 'none') {
+    mobileNav.style.display = 'block';
+    menu.src = 'images/close-icon.png';
+  } else {
+    mobileNav.style.display = 'none';
+    menu.src = 'images/menu-icon.png';
+  }
+}
+
+menu.addEventListener('click', toggleMenu);
```
Comment (medaminedev66) [inline script.js:15]: [OPTIONAL] The user has to click twice on the hamburger menu to open it. I will recommend fixing that to improve the user experience.

**Review by medaminedev66 — APPROVED** [review]
# Hi @carlylechia 😃, 

## Status: Approved ✔️

:+1: You have implemented all the suggestions. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


**You may close this Pull Request and merge the changes**

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Amine Snahi_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 4 above)
Comment (carlylechia) [reply script.js:15]: I'll work on the logic, thank you :)

**Pull request merged**